views/leaderboard_view.py

```diff
@@ -31,34 +33,41 @@ def __add_widgets(self):
             Image.open("assets/home.png"))
 
         sort_asc_button = PageButton(
-            self, text="ASC", image=self.up_arrow_img, command=self.controller.sort_data_asc)
+            self, text="ASC", image=self.up_arrow_img, command=self.controller.sort_score_data_asc)
+        sort_nick_asc_button = PageButton(
+            self, text="ASC-NICK", image=self.up_arrow_img, command=self.controller.sort_nick_data_asc, background="#694948")
         home_button = PageButton(
             self, text="Home", image=self.home_img, command=self.controller.back_to_main)
         sort_desc_button = PageButton(
-            self, text="DESC", image=self.down_arrow_img, command=self.controller.sort_data_desc)
+            self, text="DESC", image=self.down_arrow_img, command=self.controller.sort_score_data_desc)
+        sort_nick_button = PageButton(
```

Comment:
maybe change it to `sort_nick_desc_button` to follow the same rule

Reply:
outdated